.github/workflows/deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - develop
+      - release
```

Comment:
The release branch will have a date appended to it

```suggestion
      - release-*
```

Reply:
(discussed on slack)